### PR TITLE
Issue #2 solved: Stripe API keys are no longer hard-coded

### PR DESCRIPTION
**Pull request by jose-donato**
**Expected:**
Keys to be loaded from a secure place.

The two keys that I found: STORAGE_KEY & STRIPE_API_KEY were renamed to GATSBY_STORAGE_KEY & GATSBY_STRIPE_API_KEY respectively and can be accessed with process.env.GATSBY_STORAGE_KEY or process.env.GATSBY_STRIPE_API_KEY.

Following the [Gatsby Docs](https://www.gatsbyjs.org/docs/environment-variables/), I added a file named .env.development in the root folder with the needed keys:

`GATSBY_STORAGE_KEY="ABC"`
`GATSBY_STRIPE_API_KEY="ABC"`

I renamed the variables because in gatsby to access the env variables on the client-side JS they need to be prefixed with **GATSBY_**.

### CODE REVIEW

File: snippets/lambda.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // https://stripe.com/docs/payments/without-card-authentication
-const stripe = require("stripe")("API_KEY")
+const stripe = require("stripe")(process.env.GATSBY_STRIPE_API_KEY) //"API_KEY"
```
Comment (thorsten-stripe) [inline snippets/lambda.js]: Note that this doesn't run in the context of Gatsby, so might be worthwhile just changing to `process.env.STRIPE_SECRET_KEY`

**Comment by thorsten-stripe** [comment]
Thanks @jose-donato for getting this started 👍 

All `.env*` files are being ignored in the gitignore, which is good, but it would be good to have a .example.env file that lists the var names that are being set without the actual values.

Also, in the Gatsby docs: https://www.gatsbyjs.org/docs/environment-variables/#server-side-nodejs

Project environment variables that you defined in the .env.* files will NOT be immediately available in your Node.js scripts. To use those variables, use NPM package dotenv to examine the active .env.* file and attached those values, It’s already a dependency of Gatsby, so you can require it in your gatsby-config.js or gatsby-node.js like this:
```
require("dotenv").config({
  path: `.env.${process.env.NODE_ENV}`,
})
```

So we'll need to add that in the gatsby-config file.

Lastly, we should include the publishable Stripe key also: https://github.com/jamstack-cms/jamstack-ecommerce/blob/master/src/pages/checkout.js#L20

Let me know if you need any help with these. There's also a Gatsby eCom Tutorial that outlines these steps in case helpful for others :) https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/#advanced-import-skus-via-source-plugin

File: snippets/lambda.js (hunk 1 above)
Comment (jose-donato) [reply snippets/lambda.js]: Fixed in the next pull request, should I rename GATSBY_STORAGE_KEY to STORAGE_KEY also?

**Comment by jose-donato** [comment]
Didn't saw your answer, sorry. 
Already fixed what you suggested in this last commit. 

> Lastly, we should include the publishable Stripe key also: https://github.com/jamstack-cms/jamstack-ecommerce/blob/master/src/pages/checkout.js#L20

I was a bit confused after reading this, so I added an additional STRIPE_PUBLISHABLE_KEY, should it be the same as STRIPE_SECRET_KEY but called in both lambda.js and checkout.js or they are different keys?

**Pull request closed without merging**